service-front/module/Application/src/Controller/General/CookiesController.php

```diff
@@ -23,16 +22,20 @@ public function indexAction()
         $cookiePolicy = $this->fetchPolicyCookie($request);
 
         if ($request->isPost()) {
+            $this->flashMessenger()->addSuccessMessage('You’ve set your cookie preferences.');
             $form->setData($request->getPost());
 
-            $cookiePolicyViewed = new SetCookie(self::SUBMITTED_COOKIE_PAGE);
-            $cookiePolicyViewed->setValue('true')
-                ->setHttponly(false)
-                ->setSecure(true)
-                ->setExpires(new \DateTime('+60 seconds'));
-            $this->getResponse()->getHeaders()->addHeaderLine($cookiePolicyViewed->getFieldName(), $cookiePolicyViewed->getFieldValue());
+            if ($form->get('usageCookies')->getValue() === 'yes') {
+                $cookiePolicy['usage'] = true;
+            } else {
+                $cookiePolicy['usage'] = false;
 
-            $cookiePolicy['usage'] = $form->get('usageCookies')->getValue() === 'yes' ? true : false;
```

Comment:
Do we need to leave this line in place? I think this is how the form decides which radio button to set.